txeventq/kafka-connector-example/sql/setupTeq.sql

```diff
@@ -0,0 +1,25 @@
+set serveroutput on
+create user if not exists &1 identified by &2
+
+alter pluggable database all close immediate;
+drop pluggable database cdb1_pdb1 including datafiles;
+drop pluggable database cdb1_pdb2 including datafiles;
+drop pluggable database cdb1_pdb3 including datafiles;
+drop pluggable database cdb1_pdb4 including datafiles;
+create pluggable database cdb1_pdb1 &1 user &1 identified by  &2 file_name_convert=('pdb0', 'pdb1');
```

Comment:
Against which database is this sample supposed to run? There are no off-the-shelf on-prem databases that ship a `pdb0` nor `cdb1_pdb1`, `cdb1_pdb2`, `cdb1_pdb3`, `cdb1_pdb4` so most of these statements will fail, most likely also the `cdb1_pdb1` creation statement.

Reply:
I modified the sql script to use an additional parameter for the name of the seed database. The seed database name that will be used will be passed into the initdb.sh script.